Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -134,40 +135,53 @@ def incident_data_to_demisto_format(inc_data):
     return formatted_data
 
 
-def get_update_incident_request_data(client, args):
+def get_update_incident_request_data(client: Client, args: Dict[str, str]):
     # Get Etag and other mandatory properties (title, severity, status) for update_incident command
-    _, _, result = get_incident_by_id_command(client, args)
+    _, _, fetched_incident_data = get_incident_by_id_command(client, args)
 
     title = args.get('title')
     description = args.get('description')
     severity = args.get('severity')
     status = args.get('status')
     classification = args.get('classification')
     classification_reason = args.get('classification_reason')
+    assignee_email = args.get('assignee_email')
+    labels = argToList(args.get('labels', ''))
 
     if not title:
-        title = result.get('properties', {}).get('title')
+        title = demisto.get(fetched_incident_data, 'properties.title')
     if not description:
-        description = result.get('properties', {}).get('description')
+        description = demisto.get(fetched_incident_data, 'properties.description')
     if not severity:
-        severity = result.get('properties', {}).get('severity')
+        severity = demisto.get(fetched_incident_data, 'properties.severity')
     if not status:
-        status = result.get('properties', {}).get('status')
+        status = demisto.get(fetched_incident_data, 'properties.status')
+    if not assignee_email:
+        assignee_email = demisto.get(fetched_incident_data, 'properties.owner.email')
 
-    inc_data = {
-        'etag': result.get('etag'),
+    existing_labels = demisto.get(fetched_incident_data, 'properties.labels')
+    if not labels:  # not provided as arg
+        labels_formatted = existing_labels
+
+    else:
+        labels_formatted = [{"labelName": label, "labelType": "User"}
+                            for label in argToList(labels) if label]  # labels can not be blank
```

Comment:
what about the existing labels? does this override them?

Reply:
Yes, as that's what happens on a regular API call.